appcues/src/main/java/com/appcues/debugger/DebuggerRecentEventsManager.kt

```diff
@@ -178,3 +185,32 @@ internal class DebuggerRecentEventsManager(
         _data.emit(list)
     }
 }
+
+private fun Map<String, Any>.filterOutScreenProperties(eventType: EventType): Map<String, Any> {
+    return if (eventType == EventType.SCREEN) {
+        filterNot { it.key == ActivityRequestBuilder.SCREEN_TITLE_ATTRIBUTE }
+    } else this
+}
+
+private fun Map<String, Any>.filterOutAutoProperties(): Map<String, Any> {
+    return filterNot { it.key == AutoPropertyDecorator.IDENTITY_PROPERTY }
+}
+
+private fun Map<String, Any>.getAutoProperties(): Map<String, Any> {
+    @Suppress("UNCHECKED_CAST")
+    return (this[AutoPropertyDecorator.IDENTITY_PROPERTY] as Map<String, Any>?) ?: mapOf()
+}
+
+private fun Map<String, Any>.filterOutInteractionData(): Map<String, Any> {
+    return filterNot { it.key == ExperienceLifecycleEvent.INTERACTION_DATA_KEY }
+}
+
+private fun Map<String, Any>.getFormResponse(): Map<String, Any?> {
+    return (this[ExperienceLifecycleEvent.INTERACTION_DATA_KEY] as? ExperienceStepFormState)?.let {
+        it.formItems.associate { itemState ->
+            itemState.label to itemState.value
+        }
+    } ?: mapOf()
+}
+
+private fun List<Pair<String, Any?>>.sortByPropertyName(): List<Pair<String, Any?>> = sortedBy { it.first }
```

Comment:
these helpers got moved down to the file level and not nested in the class to make Detekt happy - it does not like having too many functions in a class.  These are not really functions of the class but just extension helpers that get used, so moved them out.